salt/states/win_system.py

```diff
@@ -115,7 +115,7 @@ def computer_name(name):
     if result is not False:
         after_name = result['Computer Name']['Current']
         after_pending = result['Computer Name'].get('Pending')
-        if ((after_pending is not None and after_pending == name) or
+        if ((after_pending is not None and after_pending == name.upper()) or
                 (after_pending is None and after_name == name)):
```

Comment:
You removed the `.upper()` for `name` in L91 and put it back in on L118 but not in the comparison in L119. Is this a bug?